site/src/job_queue/mod.rs

```diff
@@ -381,4 +456,30 @@ mod tests {
         })
         .await;
     }
+
+    #[test]
+    fn create_benchmark_jobs_default() {
```

Comment:
I don't think that this test is super useful, tbh. I'd just call `create_benchmark_jobs` to actually populate the DB, and then use DB functions to load jobs from the DB and assert their properties. We don't have the loading functions yet, so either you can add them in this PR and keep the test, or I'd remove the test and only add it once we can actually check what happened in the database.